plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentExt.kt

```diff
@@ -58,4 +59,44 @@ fun LocationComponentPlugin.createDefault2DPuck(
       )
     )
   }
+}
+
+/**
+ * Adds a listener that gets invoked when indicator accuracy radius changes.
+ *
+ * @param listener Listener that gets invoked when indicator accuracy radius changes
+ */
+fun LocationComponentPlugin.addOnIndicatorAccuracyRadiusChangedListener(listener: OnIndicatorAccuracyRadiusChangedListener) {
+  (this as LocationIndicatorAccuracyRadiusHandler).addOnIndicatorAccuracyRadiusChangedListener(listener)
```

Comment:
Would we like to do safe cast `(this as? LocationIndicatorAccuracyRadiusHandler)?.addOnIndicatorAccuracyRadiusChangedListener(listener)` instead? 
It may be fine even throwing an exception to user if LocationComponent is NOT our impl but custom one. We just need clear message, explaining why method throws.

Reply:
`LocationIndicatorAccuracyRadiusHandler` is an interface, it will work fine even LocationComponent is a custom one as long as it implements this interface.

Reply:
Right, but if user does not implement the interface? Seem that can easily happen, many users with updated SDK who has custom plugin implementations (not sure how common is it though) most probably won't implement it. 
Then when this cast crashes it won't give many useful information about the reason. Should we throw an exception with the clear message, like `your custom plugin implementation does not implement the necessary interface` ?

Reply:
Right, let's add a check for this and this should also fix @kiryldz 's question about API breaking.

Reply:
Just not sure if this fixes @kiryldz comment, I've had an imression that only LocationPluginImpl should implement the new interface, otherwise it indeed seems a breaking change